### PR TITLE
tests: fix ovn command arguments

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-orientdb

File: tests/tests.go [hunk 1]
```diff
@@ -290,7 +290,12 @@ func initConfig(conf string, params ...helperParams) error {
 
 func execCmds(t *testing.T, cmds ...Cmd) (e error) {
 	for _, cmd := range cmds {
-		args := strings.Split(cmd.Cmd, " ")
+		// ugly workaround to handle arguments that contain spaces, these need to be escaped with \\
```
Comment (safchain) [inline tests/tests.go]: what about doing something like this
https://play.golang.org/p/YRRnYC-UDqA

A kind of real command line parser

Comment (hunchback) [reply tests/tests.go]: You need somthing which mimic shell handling - for that you may consider:

https://github.com/kballard/go-shellquote

**Pull request merged**